<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>citieslist</class>
 <widget class="QDialog" name="citieslist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cities list</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="CityInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Locality</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Postal Code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INSEE Code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Departement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>District</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Country</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label00">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>140</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Input place</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="CitySelect">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="TransfertIn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ressources_png/fleche1.jpg</normaloff>ressources_png/fleche1.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="TransfertOut">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ressources_png/fleche2.jpg</normaloff>ressources_png/fleche2.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChooseFile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>citieslist</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>citieslist</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
